pkg/utils/distro/distro.go

```diff
@@ -14,12 +14,26 @@
 
 package distro
 
-var data map[string]interface{}
+import (
+	"sync/atomic"
 
-func Replace(distro map[string]interface{}) {
-	data = distro
+	"github.com/pingcap/log"
+	"go.uber.org/zap"
+)
+
+type introData map[string]interface{}
+
+var data atomic.Value
+
+func Replace(distro introData) {
+	data.Store(distro)
 }
 
 func Data(k string) string {
-	return data[k].(string)
+	d := data.Load().(introData)
+	if d[k] == nil {
+		log.Info("Distro information doesn't exist", zap.String("key", k))
```

Comment:
Maybe we can remove this log! Since we can just know the missing key from the output.

Reply:
Yeah, let's go!